numpy/_core/include/numpy/npy_common.h

```diff
@@ -168,6 +168,9 @@
     #define npy_ftell ftell
 #endif
     #include <sys/types.h>
+    #ifndef _WIN32
+        #include <unistd.h>
+    #endif
```

Comment:
Interesting, seems like the windows `<io.h>` equivalence is still in python, I guess.  (or maybe I misunderstand).

Anyway, LGTM, thanks.  If something is odd on `_WIN32` here we would notice it anyway.

Reply:
CI failed for 32-bit Python on Windows. I checked in a few other libraries like `zlib` and `blosc`, and they all protect this include like that, so it should be fine.

Reply:
Right, was just waiting for CI, thanks.  It seems to me that `lseek` on windows may need `<io.h>` but Python probably still has that.

Reply:
Guess we'll find out if and when they clean it up:)